Migrate quality UI changes (#7441)
opencv/cvat: changelog.d/20240206_125854_klakhov_quality_bugfixes.md

### Fixed

> - On task quality page only the first job page had quality report results displayed in UI

Reviewer: ```suggestion
- On quality page for a task, only the first page with jobs has quality report metrics
```
